integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/ui/portal/PortalFragment.kt

```diff
@@ -319,7 +320,10 @@ class PortalFragment : Fragment(R.layout.fragment_portal) {
 
     private fun showConfirmationDialog() {
         viewModel.logEvent(AnalyticsConstants.CrowdNode.PORTAL_VERIFY)
-        ConfirmationDialog().show(parentFragmentManager, "confirmation_dialog")
+
+        if (requireActivity().lifecycle.currentState.isAtLeast(Lifecycle.State.RESUMED)) {
```

Comment:
https://console.firebase.google.com/u/1/project/dash-wallet-firebase/crashlytics/app/android:hashengineering.darkcoin.wallet/issues/80cc916b07be47b7b1c0bf105ec1f1d4?time=last-seven-days&types=crash&sessionEventKey=62B551EF02A7000109F896B3C276065A_1691134087670467691

Maybe it'll be moved into the OffsetDialog itself later the same as with AdaptiveDialog.